diff -uNr Lua-cURLv3-0.3.8/src/lceasy.h Lua-cURLv3-0.3.8_patched/src/lceasy.h
--- Lua-cURLv3-0.3.8/src/lceasy.h	2018-05-13 12:06:46.000000000 +0200
+++ Lua-cURLv3-0.3.8_patched/src/lceasy.h	2018-06-23 22:05:54.592621142 +0200
@@ -37,6 +37,10 @@
 
 #define LCURL_EASY_MAGIC 0xEA
 
+#if defined(__GNUC__) && __GNUC__<5
+#       include "lcmulti.h"
+#       include "lcmime.h"
+#else
 #if LCURL_CC_SUPPORT_FORWARD_TYPEDEF
 typedef struct lcurl_multi_tag lcurl_multi_t;
 #if LCURL_CURL_VER_GE(7,56,0)
@@ -50,6 +54,7 @@
 #define lcurl_mime_t struct lcurl_mime_tag;
 #endif
 #endif
+#endif
 
 typedef struct lcurl_easy_tag{
   unsigned char magic;
